### DISCUSSION TITLE
Copilot setting document

### DISCUSSION THREAD

**Discussion by tszdanger** [community/community#19537] [Copilot Conversations] [answered]
Hi,

I'm trying the different settings for Github Copilot.

I've found that we can configure a parameter named **github.copilot.advanced**.

In this parameter, we have the following choices:

"debug.testOverrideProxyUrl":Override GitHub authentication proxy URL when running tests

What does this mean? Why do we need to override GitHub authentication proxy URL? Can anyone explain this to me? I would appreciate it :) .

Thanks!

**Comment by davecheney** [comment, marked as answer]
It is used for debugging purposes while developing copilot-proxy.githubusercontent.com.

**Reply by tszdanger** [reply]
Thanks for answering!
